reframe/core/buildsystems.py

```diff
@@ -655,6 +655,35 @@ def emit_build_commands(self, environ):
         return prepare_cmd + [' '.join(configure_cmd), ' '.join(make_cmd)]
 
 
+class EasyBuild(BuildSystem):
+    easyconfigs = fields.TypedField(typ.List[str])
+    options = fields.TypedField(typ.List[str])
+
+    def __init__(self):
+        super().__init__()
+        self.easyconfigs = []
+        self.options = []
+        self._installpath = None
+
+    def emit_build_commands(self, environ):
+        _easyconfigs = ' '.join(self.easyconfigs)
+        _options = ' '.join(self.options)
+        return [f"eb {_easyconfigs} --installpath {self._installpath} "
+                f"--tmpdir {self._installpath} "
+                f"--sourcepath {self._installpath} "
+                f"--buildpath {self._installpath}/build {_options}"]
```

Comment:
Very often the installation directory and the build directory are different filesystem (like NFS for install dir, `/dev/shm` for build dir), so maybe you also need `self._buildpath`?

Reply:
Users can pass any argument to `eb` through the build system's `options`.